docs/build.js

```diff
@@ -14,23 +14,33 @@ const license = path.join(repoRoot, 'LICENSE');
 const readmeSrc = path.join(__dirname, 'README.docs.md');
 const readmeDest = path.join(docsBuilt, 'README.md');
 
+/**
+ * Generates HTML code for `fileName` page.
+ *
+ * @param {string} fileName Path for Router.Route
+ * @return {Promise} promise
+ * @internal
+ */
+function generateHTML(fileName) {
+  return new Promise((resolve, reject) => {
+    Router.run(routes, '/' + fileName, Handler => {
+      let html = React.renderToString(React.createElement(Handler));
+      html = '<!doctype html>' + html;
```

Comment:
It was originally in lowercase https://github.com/react-bootstrap/react-bootstrap/blob/120ba6d38a49b7a17c1418aab8fc2e44ac09c8b0/docs/src/Root.js#L45, so I just return it back :smiley: 
And I'm looking at `source` tab in browser, all html code is lowercased.
The same `style` :smile: